semantics.json

```
@@ -142,7 +142,7 @@
       {
         "label": "Language mode label",
         "name": "languageModeLabel",
-        "default": "Change language mode",
+        "default": "Swap language",
```

Swap language or swap languages?

```suggestion
        "default": "Swap languages",
```

---

Good question 🤔 What do you think?

---

Used languages

---

I think maybe plural as we're swapping to things